Source/Editor/Windows/Assets/JsonAssetWindow.cs

```diff
@@ -139,6 +191,29 @@ protected override void OnAssetLoaded()
 
             if (_typeText != null)
                 _typeText.Dispose();
+
+            // Get content item for options button
+            object buttonTag = null;
+            var allTypes = Editor.CodeEditing.All.Get();
+            foreach (var type in allTypes)
+            {
+                if (type.TypeName.Equals(Asset.DataTypeName, StringComparison.Ordinal))
+                {
+                    buttonTag = type.ContentItem;
+                    break;
+                }
+            }
+            
+            _optionsButton = new ToolStripButton(_toolstrip.ItemsHeight, ref Editor.Icons.Settings12)
+            {
+                AnchorPreset = AnchorPresets.TopRight,
+                Tag = buttonTag,
```

Comment:
`Size = new Float2(18),` would make the that icon look lees pixalated

Reply:
Doing this does make it smaller than the other toolstrip icons. I guess because it is off in the corner, it does not look bad being smaller.